components/org.wso2.carbon.identity.idp.metadata.saml2/src/main/java/org/wso2/carbon/identity/idp/metadata/saml2/util/SAML2SSOFederatedAuthenticatorConfigBuilder.java

```diff
@@ -62,260 +80,261 @@ private static FederatedAuthenticatorConfig parse(EntityDescriptor entityDescrip
+                }
+                properties[2] = property;
+
+                property = new Property();
+                property.setName(IdentityApplicationConstants.Authenticator.SAML2SSO.IS_AUTHN_REQ_SIGNED);
+                if (idpssoDescriptor.getWantAuthnRequestsSigned() != null && idpssoDescriptor.getWantAuthnRequestsSigned()) {
+                    property.setValue("true");
+                } else {
+                    property.setValue("false");
+                }
+                properties[3] = property;
+
+                property = new Property();
+                property.setName(IdentityApplicationConstants.Authenticator.SAML2SSO.IS_LOGOUT_ENABLED);
+                List<SingleLogoutService> singleLogoutServices = idpssoDescriptor.getSingleLogoutServices();
+                if (CollectionUtils.isNotEmpty(singleLogoutServices)) {
+                    property.setValue("true");
+                } else {
+                    property.setValue("false");
+                }
+                properties[4] = property;
+
+                property = new Property();
+                property.setName(IdentityApplicationConstants.Authenticator.SAML2SSO.LOGOUT_REQ_URL);
+                if (CollectionUtils.isNotEmpty(singleLogoutServices)) {
+                    boolean foundSingleLogoutServicePostBinding = false;
+                    for (SingleLogoutService singleLogoutService : singleLogoutServices) {
+                        if (singleLogoutService != null) {
+                            if (singleLogoutService.getBinding() != null && singleLogoutService.getBinding().equals(SAMLConstants.SAML2_POST_BINDING_URI) && singleLogoutService.getLocation() != null) {
+                                property.setValue(singleLogoutService.getLocation());
+                                foundSingleLogoutServicePostBinding = true;
+                                break;
+                            }
+                        }
                     }
-                    properties[4] = property;
-
-                    property = new Property();
-                    property.setName(IdentityApplicationConstants.Authenticator.SAML2SSO.LOGOUT_REQ_URL);
-                    if (CollectionUtils.isNotEmpty(singleLogoutServices)) {
-                        boolean foundSingleLogoutServicePostBinding = false;
+                    if (!foundSingleLogoutServicePostBinding) {
                         for (SingleLogoutService singleLogoutService : singleLogoutServices) {
                             if (singleLogoutService != null) {
-                                if (singleLogoutService.getBinding() != null && singleLogoutService.getBinding().equals(SAMLConstants.SAML2_POST_BINDING_URI) && singleLogoutService.getLocation() != null) {
+                                if (singleLogoutService.getBinding() != null && singleLogoutService.getLocation() != null) {
```

Comment:
Any particular reason for removing this?
`singleLogoutService.getBinding().equals(SAMLConstants.SAML2_POST_BINDING_URI)`

Reply:
This has not change and it is detected since the file lines changed drastically.